bin/ember-template-lint.js

```diff
@@ -172,6 +191,13 @@ function run() {
     }
   }
 
+  if (printPending && filesWithErrors.length > 0) {
+    console.log(
+      'Add the following to your `.template-lintrc.js` file to mark these files as pending.\n\n'
+    );
+    console.log(`pending: ${JSON.stringify(filesWithErrors, null, 2)}`);
+  }
+
   if (Object.keys(errors).length) {
```

Comment:
I don't think we should emit the actual errors in addition to printing the pending list when `--print-pending` is used. Can you tweak things to avoid that (I'm thinking of a `return;` added to the `if` just above)...